definitions/synth-runtime_validation/summary_metrics.yml

```diff
@@ -0,0 +1,20 @@
+monitorName:
```

Comment:
Could we change this key to `name` instead of `monitorName`. This request is two fold, the `EntityTableRowCell` component expects a `name` field on the entity in order to render correctly. We could do some transforming on the client but if we could fix this on the server, even better. Also, I noticed that the current entities in the current monitor list have a `name` field so this would be consistent with that implementation.

Reply:
updated field but left title the same. Is this alright with you @codymjarrett ?

Reply:
Thanks for that. Honestly, I don't know too much about how entities are created here. It's interesting because looking at the [entity definition for the current monitor list](https://github.com/newrelic/entity-definitions/blob/main/definitions/synth-monitor/definition.yml) there isn't a `name` or `monitorName` field defined however there's a `name` field on the entity objects returns from the response. Is there some magic happening that isn't explicitly detailed here?  @mderemer-nr

Reply:
@codymjarrett that is because the entity is not created via synthesis. If it were, then you'd have to define how to map a specific attribute from the telemetry to a specific entity field. 

As this is not the case for this entity, it will be up to the (New Relic internal) system that generates those entities to specify the value for those fields.